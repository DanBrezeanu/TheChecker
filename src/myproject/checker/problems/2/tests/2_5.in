531179462 462410036 272843460 270488152 127276031 170630883 415935505 758163929 346526586 308141184 296385881 292055848 921653071 806433853 972366582 230939332 198281811 466668312 266009315 850944663 755323166 57154223 738196619 447182690 443673151 483346884 914819795 701278898 395058566 237514685 495174312 453328672 211714507 75133414 329555907 13485163 345409435 425881382 319781863 742004884 299501767 44560070 964851291 273819606 449291485 900630834 623862391 864704246 19794795 195226950 245990107 727836149 835623857 184973946 803203601 560499627 182413905 700745246 75511961 162815961 633227943 61049765 185030950 376867979 841026104 203636533 724624229 198499302 407116317 85234391 450067878 888488265 888130030 159649632 1664147 121815990 807251325 306662102 939319821 904677656 857632111 961390987 321053732 355431353 8545764 453851960 740586278 892648457 752405104 752444989 690801294 522847374 562243307 273886726 267739795 654499903 601963014 768547192 827429489 934869896 964222012 597768481 75199170 280238558 38999422 587494753 543118825 725115689 849293025 994535770 55765111 286082413 240903563 151657025 83914452 210236882 948561801 687610034 369222387 555370365 766869312 8098384 856792679 983515593 549990612 920572038 925486715 903218251 127250477 210035467 783090638 725695441 803725842 81488186 168424284 694546967 843522177 161440398 691705555 197469570 239030101 791650257 672019256 137998872 941323748 767826324 544765478 368434269 144302959 447637612 428427863 258707734 241282250 701686047 777166236 167642402 716236255 468831475 148648848 396866596 309242659 159774161 116555834 672226362 294114000 189203222 694297275 211319226 477041512 838836755 719632818 11288077 210511788 708152502 21816825 908103367 156970253 860704008 188067644 681037663 582915076 953027099 113071623 524969130 688057264 442386438 615199214 301073989 599323404 590928445 290345034 922283334 288880533 22151851 85312643 182116030 88507894 94124305 42103293 57167284 153684865 834323088 102359890 925179902 464818809 353706519 541349982 769783991 195404060 614338453 149447950 655736768 259702776 713396095 424434963 30448863 993227919 432395896 979063798 839904159 138517045 702095419 222943157 491508755 362042629 962194091 345655383 373748914 388922264 158181579 924377837 143440257 670248619 165492759 72649224 771683882 301839279 201268377 472501643 414190628 604360272 805609414 417745711 295553438 535997848 25314722 334699601 150116636 207518134 164591209 350791890 696236629 226750922 37612117 27793314 716616128 592861788 129111001 151163967 549096996 867576084 313181858 246115830 758614457 524251776 167469259 570111036 577193502 595190571 886982853 438816409 517499944 34837725 691434975 152731978 726442598 681040463 365015400 92569888 60468494 639902324 152043822 328319157 999659721 126014716 691749774 13717595 711079539 317587182 985459447 744572711 326690676 599602821 464278448 749672836 550595484 142884603 76741952 623663510 63029280 845782979 572311717 91742444 321246914 95454502 169031918 525184173 957222470 568706568 732392848 22605786 352210551 299440503 268032388 699849282 491189067 612407631 842880952 22144562 254214537 370237412 848580844 720789312 237385223 730430998 966918002 546917141 732255921 692700942 285069116 250752817 730947316 366286644 84346963 373574053 993962924 969613859 197933993 667105523 388551331 996370169 716979597 646221899 510344476 19957531 716330842 517201702 983823947 525763326 143213735 96483938 149357936 639256245 822547661 979522739 217390521 981578952 713821632 435137998 145955708 954574571 870271617 843551601 481609383 350814799 990909702 285390185 205178359 31217146 709737303 729833237 295546775 150777301 184312247 857622834 278058550 639645327 833342559 906275216 257194350 998733371 932773882 649077069 243505267 766471951 575056009 91167 561312467 740203857 336772733 791783322 579512057 533575157 255229990 134792514 530030790 275491363 212684588 341798620 473771320 412254028 845065165 722976761 98590059 38811987 977905776 40316464 842374128 538567070 787899546 191586996 222071139 179841445 464776572 412926922 473996053 758619651 411251075 712465426 122745031 788449736 954835227 704544236 604095556 989377459 215397336 936955973 218137920 175550933 741603811 177208626 561105351 629456944 716478841 785056229 840089293 769433007 979796252 167535825 504419748 741783935 598665525 723714512 403497666 537361159 952066904 109091657 745104803 960737562 565805935 516293115 365558649 740225170 73843159 197954911 421339803 212347505 822039456 136427144 351639853 898012904 385155052 7440046 564194628 492687074 583807375 217260673 15084217 712260359 959419379 702065452 6098053 850090580 409510555 277066345 264015726 354926982 212100962 551976760 993686625 525010562 982440918 979626837 716178193 657620221 839395932 83833519 163788936 188173809 63134339 949431937 103871349 633588753 563074065 922975571 490336208 530077538 196225235 331313585 497215714 300665618 748040165 867631650 787931488 770706614 848551479 146613938 163670015 220391629 469327400 630375714 708403060 897403645 115252812 36460033 229206843 295057976 999023862 214105809 268132052 820551223 454695872 963322640 486360152 123164540 40075203 755353874 253621483 465123943 472926612 857007686 400049222 857927219 163808532 550672774 320029416 801939573 501289003 670176910 464060237 467293919 105282830 508586563 919972953 671638639 479031119 864767503 91210439 883865533 299355130 822321508 843096166 218486225 745470627 849104735 303186787 736571744 516250729 54591743 369025669 186680861 389768573 68608448 897529462 138931301 110509704 221538508 841395884 274351036 637803880 917743580 869075822 807935559 705120709 437141349 391582142 832696051 576898766 318942292 312284540 797967415 212184840 489426055 437477773 756813203 619250607 113414794 388927581 452197724 970212438 428655399 125374956 169337593 336511991 536240305 638497149 981258421 372979484 417379369 285487725 199699143 672560198 401025012 446921348 329486542 455607737 165824505 399093966 347669631 245797991 494424440 722474415 247929345 59191879 345943477 366758164 836407492 253747996 800616339 292391822 287232880 298312249 411308718 122150780 869571974 986079128 924237632 883729343 242294149 682952445 254635287 377740935 936208419 666091758 869799758 621680472 650976095 814016215 44538782 183768436 875113786 169798500 581955764 582814509 817169115 736599228 199404212 21844557 806219400 238056757 278486722 178765499 105287053 680682018 338186590 556721603 437213696 421790315 727508270 747266694 982956236 244300899 973214805 749034021 574364092 123644553 601757434 219755287 439786250 681812131 530028644 48368095 312501930 497819128 911354486 324103457 173064206 275091168 139090537 317865960 615156824 771423558 34295713 112232958 335482119 82998846 149224545 470039104 681607576 996532058 165758884 82824944 832237795 565866553 399981549 671661837 999087889 985296950 487168036 16798679 989296399 188604150 193503847 995556208 941954910 289701582 447351370 217589875 657802645 669207622 698890038 573752086 296420827 709199744 304485318 538408696 815849903 357469948 60802513 438990969 796020300 247459264 272804458 536077263 888975207 931283899 114593168 263041497 573583992 403927497 360804961 39485363 530176852 395239895 285302685 841369316 15832920 359045154 549716050 240398339 858303670 332642727 677513237 417273823 107689410 623661599 328249680 105681733 603255953 19298747 444865427 710135021 266129685 262774059 826685798 356108045 52018757 127480623 417365946 45155270 389330837 353572820 513198367 430147267 336450395 7857900 825335197 90994575 877206233 724433110 74529924 825850700 298727549 330576854 207223355 870868506 552428662 360112293 778231503 30247381 828055327 60276064 373743301 553038258 662260607 864012129 73732698 236704701 809729276 586522444 545107389 485852049 24855360 36740818 619225037 217949134 186047021 486862409 204240265 412765548 150736509 86640828 253288070 207375957 434915125 94535873 399146117 331378495 416972180 20290287 247389139 525335422 431163291 391170898 388133551 370682853 578341403 796885307 262893418 765936206 862527266 781897935 704109646 218266639 805499654 611802255 212325924 832046273 73224080 924803851 55314371 861466549 37195276 634952858 947646901 726625031 436030605 909729240 429595635 200574039 851658435 276829253 433134754 63283333 982185987 263121475 474668849 659425954 349079291 925904178 461951885 623655025 58530569 139399633 628078139 454141821 938489397 930341274 440280353 83463707 181941661 576281671 0
