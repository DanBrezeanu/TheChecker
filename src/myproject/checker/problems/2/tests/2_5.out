482963770.50
